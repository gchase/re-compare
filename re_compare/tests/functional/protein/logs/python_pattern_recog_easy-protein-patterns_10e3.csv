regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,0.0629425048828125,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,0.0629425048828125,"[-1, -1]"
DHKNLD.,first_match,0,0.0030994415283203125,"[-1, -1]"
DHKNLD.,all_matches,0,0.0030994415283203125,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,0,0.041961669921875,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,0,0.041961669921875,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.008106231689453125,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.008106231689453125,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.00286102294921875,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.00286102294921875,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0019073486328125,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.00476837158203125,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,0,0.07414817810058594,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,0,0.07414817810058594,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.00476837158203125,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.00476837158203125,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.0059604644775390625,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.010728836059570312,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,0,0.03886222839355469,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,0,0.03886222839355469,"[-1, -1]"
P..L...KWE.,first_match,0,0.0050067901611328125,"[-1, -1]"
P..L...KWE.,all_matches,0,0.0050067901611328125,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,0.03504753112792969,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,0.03504753112792969,"[-1, -1]"
