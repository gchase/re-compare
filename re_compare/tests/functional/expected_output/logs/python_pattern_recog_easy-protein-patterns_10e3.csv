regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,0.04673004150390625,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,0.04673004150390625,"[-1, -1]"
DHKNLD.,first_match,0,0.0040531158447265625,"[-1, -1]"
DHKNLD.,all_matches,0,0.0040531158447265625,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,0,0.03409385681152344,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,0,0.03409385681152344,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.00476837158203125,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.00476837158203125,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.0030994415283203125,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.0030994415283203125,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0019073486328125,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.0050067901611328125,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,0,0.06413459777832031,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,0,0.06413459777832031,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.00476837158203125,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.00476837158203125,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.006198883056640625,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.010967254638671875,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,0,0.06508827209472656,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,0,0.06508827209472656,"[-1, -1]"
P..L...KWE.,first_match,0,0.0040531158447265625,"[-1, -1]"
P..L...KWE.,all_matches,0,0.0040531158447265625,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,0.025033950805664062,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,0.025033950805664062,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,0.006198883056640625,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,0.006198883056640625,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,1,0.009059906005859375,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,0,0.048160552978515625,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,0,0.048160552978515625,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,0.00286102294921875,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,0.00286102294921875,"[-1, -1]"
A|G....G,first_match,1,0.003814697265625,"[15, 15]"
A|G....G,consecutive,1,0.003814697265625,"[15, 15]"
A|G....G,consecutive,2,0.0030994415283203125,"[19, 19]"
A|G....G,all_matches,2,0.0069141387939453125,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,0,0.186920166015625,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,0,0.186920166015625,"[-1, -1]"
